<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1178</width>
    <height>524</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Result Analyser - Performance Report Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/img/Images/Icons/MainView_Icon.png</normaloff>:/img/Images/Icons/MainView_Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDockWidget" name="dw_Central">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1180</width>
      <height>636</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1180</width>
      <height>636</height>
     </size>
    </property>
    <property name="toolTip">
     <string>Home Page</string>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="features">
     <set>QDockWidget::NoDockWidgetFeatures</set>
    </property>
    <property name="windowTitle">
     <string>Home Page</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="lo_Central_Main" stretch="0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="s_subject">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tw_Students">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>1178</width>
            <height>578</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0,0,0,)</string>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_Home">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <attribute name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/img/Images/Icons/home.png</normaloff>:/img/Images/Icons/home.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Home</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
          <widget class="QWidget" name="tab_Students">
           <attribute name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/img/Images/Icons/Students_Icon.png</normaloff>:/img/Images/Icons/Students_Icon.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Students</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_Scores">
           <attribute name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/img/Images/Icons/Score_Tab_Icon.png</normaloff>:/img/Images/Icons/Score_Tab_Icon.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Scores</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QTableView" name="tv_scores">
              <property name="font">
               <font>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>160</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_Results">
           <attribute name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/img/Images/Icons/Report_Card_Icon.png</normaloff>:/img/Images/Icons/Report_Card_Icon.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Results</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_Performance">
           <attribute name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/img/Images/Icons/MainView_Icon.png</normaloff>:/img/Images/Icons/MainView_Icon.png</iconset>
           </attribute>
           <attribute name="title">
            <string>Performance</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuPrint">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionStudents"/>
    <addaction name="actionScores"/>
    <addaction name="actionResults"/>
    <addaction name="actionPerformance"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPrint"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dw_sidebar">
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>114</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>524287</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Classes</string>
   </property>
   <property name="toolTipDuration">
    <number>4</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255,255,255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Classes</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="lo_Classes"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/new.png</normaloff>:/img/Images/Icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/open.png</normaloff>:/img/Images/Icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/close.png</normaloff>:/img/Images/Icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/exit.png</normaloff>:/img/Images/Icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/print.png</normaloff>:/img/Images/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/settings.png</normaloff>:/img/Images/Icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/undo.png</normaloff>:/img/Images/Icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/redo.png</normaloff>:/img/Images/Icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/login.png</normaloff>:/img/Images/Icons/login.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/logout.png</normaloff>:/img/Images/Icons/logout.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/home.png</normaloff>:/img/Images/Icons/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Alt+H</string>
   </property>
  </action>
  <action name="actionStudents">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/Students_Icon.png</normaloff>:/img/Images/Icons/Students_Icon.png</iconset>
   </property>
   <property name="text">
    <string>Students</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionScores">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/Score_Tab_Icon.png</normaloff>:/img/Images/Icons/Score_Tab_Icon.png</iconset>
   </property>
   <property name="text">
    <string>Scores</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionResults">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/Report_Card_Icon.png</normaloff>:/img/Images/Icons/Report_Card_Icon.png</iconset>
   </property>
   <property name="text">
    <string>Results</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
  <action name="actionPerformance">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/Images/Icons/MainView_Icon.png</normaloff>:/img/Images/Icons/MainView_Icon.png</iconset>
   </property>
   <property name="text">
    <string>Performance</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
